lib/web/integrations_awsoidc.go

```diff
@@ -857,7 +859,22 @@ func (h *Handler) awsOIDCConfigureIdP(w http.ResponseWriter, r *http.Request, p
 		return nil, trace.BadParameter("invalid role %q", role)
 	}
 
-	proxyAddr, err := oidc.IssuerFromPublicAddress(h.cfg.PublicProxyAddr)
+	s3Bucket := queryParams.Get("s3Bucket")
+	s3Prefix := queryParams.Get("s3Prefix")
+	if s3Bucket == "" || s3Prefix == "" {
+		return nil, trace.BadParameter("s3Bucket and s3Prefix query params are required")
+	}
```

Comment:
It's kind of an unsolved problem in general in the Teleport web UI, but could we make these parameters optional and fall back to `--proxy-public-url` in case a user tries to set up the integration in the middle of a proxy upgrade, or would we rather migrate to the bucket way ASAP even if users might encounter errors during the upgrade?

Reply:
I would rather have an error in those cases.
Having another set up using the proxy public url is just something that they would need to fix later on.

Now, the error won't be very friendly, but without some kind of version tracking between current loaded webapp and the server's version, I'm not sure we can improve it